1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -1,5 +1,8 @@
 
 
 export function fizzBuzz (input: number): string {
-    return "";
+    if(input === 3) {return "Fizz"; }
+    return "Buzz"
```

Comment:
Any reason to change the return type to "Buzz" in this step?

Reply:
Not really. I think I was just testing how it would return